pkg/apis/pipeline/v1beta1/pipeline_validation_test.go

```diff
@@ -2347,11 +2368,17 @@ func TestContextInvalid(t *testing.T) {
 			Params: []Param{{
 				Name: "a-param", Value: ArrayOrString{StringVal: "$(context.pipeline.missing)"},
 			}},
+			Matrix: []Param{{
+				Name: "a-param-foo", Value: ArrayOrString{ArrayVal: []string{"$(context.pipeline.missing-foo)"}},
+			}},
 		}},
-		expectedError: apis.FieldError{
+		expectedError: *apis.ErrGeneric("").Also(&apis.FieldError{
```

Comment:
what's the reason for the empty errors?

Reply:
it's needed when there are multiple errors - here's another example in existing code:
https://github.com/tektoncd/pipeline/blob/6caa150bdd14a9082fdcefec9dd374f79ba80ef4/pkg/apis/pipeline/v1beta1/pipeline_validation_test.go#L2024-L2030